### PR TITLE
feat: shutdown after certain session index for a coordinated consensu…

### PR DESCRIPTION
**Pull request by elsirion**
…s upgrade

### CODE REVIEW

File: fedimint-server/src/net/api.rs [hunk 1]
```diff
@@ -547,6 +553,15 @@ pub fn server_endpoints() -> Vec<ApiEndpoint<ConsensusApi>> {
                 Ok((&fedimint.session_status(index).await).into())
             }
         },
+        api_endpoint! {
+            SHUTDOWN_ENDPOINT,
+            ApiVersion::new(0, 0),
```
Comment (dpc) [inline fedimint-server/src/net/api.rs:558]: This ideally should be a version the call was introduced in, and I think we're at `0.2`, and should increment to 0.3. https://github.com/fedimint/fedimint/blob/bffe4660774db857ab5672f654e6eae96522246b/fedimint-server/src/config/mod.rs#L211

Comment (dpc) [reply fedimint-server/src/net/api.rs:558]: Thought being a quick EoL fix for fedimint 0.3 branch, it all probably doesn't matter.

Comment (bradleystachurski) [reply fedimint-server/src/net/api.rs:558]: If we update here, should we also update master?

Comment (dpc) [reply fedimint-server/src/net/api.rs:558]: I think so, yes. For consistency at least. I doubt it makes a difference in practice, and mostly pointing it out so in the future we don't forget.

**Comment by bradleystachurski** [comment]
Closing this PR so we can bump the api version while backporting in https://github.com/fedimint/fedimint/pull/5642

File: fedimint-server/src/net/api.rs (hunk 1 above)
Comment (bradleystachurski) [reply fedimint-server/src/net/api.rs:558]: Updating master in https://github.com/fedimint/fedimint/pull/5640

**Pull request closed without merging**